docs/dex-specification.md

```diff
@@ -192,25 +192,33 @@ The contract also manages protocol-wide charge recipient. The `feeTo` defines th
 
 <!-- github table -->
 
-|          Function           |                                Description                                |
-| --------------------------- | ------------------------------------------------------------------------- |
-| [`createPair`](#createpair) | Create a pair for two given tokens if such a pair doesn't already exist.  |
-| `getPair`                   | Return the address of the pair for two given tokens if such a pair exist. |
-| `setFeeToSetter`            | Set the new address of the protocol-wide charge recipient.                |
-| `setFeeTo`                  | Allow a provided address to control protocol-wide charge recipients.      |
-| `allPairsLength`            | Return the number of created pairs.                                       |
+|          Function           |                                                                        Description                                                                        |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`createPair`](#createpair) | Create a pair for two given tokens if such a pair doesn't already exist.                                                                                  |
+| `getPair`                   | Return the address of the pair for two given tokens if such a pair exist.                                                                                 |
+| `setFeeToSetter`            | Set the new address of the protocol-wide charge recipient. Note that a mistake in the address can result in the loss of control over the fee destination. |
+| `setFeeTo`                  | Allow a provided address to control protocol-wide charge recipients.                                                                                      |
+| `allPairsLength`            | Return the number of created pairs.                                                                                                                       |
 
 
 <!-- pdf table
 
 Function                    Description
 --------------------------- -------------------------------------------------------------------------
 [`createPair`](#createpair) Create a pair for two given tokens if such a pair doesn't already exist.  
+
 `getPair`                   Return the address of the pair for two given tokens if such a pair exist.
-`setFeeToSetter`            Set the new address of the protocol-wide charge recipient.               
+
+`setFeeToSetter`            Set the new address of the protocol-wide charge recipient.
+                            Note that a mistake in the address can result in the loss of control
+                            over the fee destination.               
+
 `setFeeTo`                  Allow a provided address to control protocol-wide charge recipients.     
```

Comment:
Can only be called by the `feeToSetter`. Sets a new address of the protocol-wide charge recipient. Setting the wrong address leads to the protocol-wide charge being lost (transferred to that wrong address).